modules/dmrpp_module/get_dmrpp/unit-tests/Makefile.am

```diff
@@ -0,0 +1,55 @@
+AUTOMAKE_OPTIONS = foreign
+HDF_FILES = *.hdf
+HDF5_FILES = *.h5
+PYTHON_FILES = *.py
+BASELINE_FILES = *.baseline
+
+EXTRA_DIST = *.hdf *.h5 *.py *.baseline
+
+cp_build_script:
+	if test $(srcdir) != .; then \
+	cp -f `echo $(srcdir) | rev | cut -c 11- | rev`/gen_dmrpp_side_car $(builddir); \
+	cp -f `echo $(srcdir) | rev | cut -c 11- | rev`/get_hdf_side_car $(builddir); \
+	cp -f `echo $(srcdir) | rev | cut -c 11- | rev`/get_dmrpp_h5 $(builddir); \
+	cp -f $(srcdir)/$(HDF_FILES) $(builddir); \
+	cp -f $(srcdir)/$(HDF5_FILES) $(builddir); \
+	cp -f $(srcdir)/$(PYTHON_FILES) $(builddir); \
+	cp -f $(srcdir)/$(BASELINE_FILES) $(builddir); \
+	cp -f $(abs_top_srcdir)/modules/dmrpp_module/build_dmrpp_h4/get_dmrpp_h4 $(builddir); \
+	cp -f $(abs_top_builddir)/modules/hdf5_handler/.libs/libhdf5_module.so $(builddir); \
+	cp -f $(abs_top_builddir)/modules/hdf4_handler/.libs/libhdf4_module.so $(builddir); \
+	cp -f $(abs_top_builddir)/modules/fileout_netcdf/.libs/libfonc_module.so $(builddir); \
+	cp -f $(abs_top_builddir)/xmlcommand/.libs/libdap_xml_module.so $(builddir); \
+	cp -f $(abs_top_builddir)/dap/.libs/libdap_module.so $(builddir); \
+	else \
+	cp -f  ../$(srcdir)/gen_dmrpp_side_car $(builddir); \
+	cp -f  ../$(srcdir)/get_hdf_side_car $(builddir); \
+	cp -f  ../$(srcdir)/get_dmrpp_h5 $(builddir); \
+	fi;
+
+check-local: cp_build_script check-local-gen-dmrpp 
```

Comment:
I was thinking about this and I _think_ it's OK, but it might be that a parallel make might build these two dependencies at the same time. That might be OK most of the time, but not always. A way to safeguard against that would be to make "cp_build_script" a dependency of the "check-local-gen-dmrpp" target. That would look like this:

```
check-local: check-local-gen-dmrpp 

check-local-gen-dmrpp: cp_build_script
    if test $(srcdir) != .; then \
    ...

```

So check-local depends on check-local-gen-dmrpp and check-local-gen-dmrpp depends on cp_build_script.

Reply:
Will do and check.